private/src/styles/utils/variables/_defaults.scss

```diff
@@ -13,33 +13,11 @@
 $enable-flexbox: true !default;
 
 /// Sets the base font size.
-$base-font-size:   18px !default;
+$base-font-size: var(--wp--preset--font-size--regular) !default;
```

Comment:
I've left `$base-font-size` defined here as it's used in the `$line-height-ratio` calculation (in the same file), and it felt odd replacing this with a CSS variable. I've removed any other references to it, excluding one other instance where it's also used in a mixin (and is also commented out).

Maybe this could be completely removed if theme.json values also replace the line-height values.